src/docfx/template/Template.cs

```diff
@@ -93,5 +139,11 @@ public static string CreateHtmlMetaTags(JObject metadata)
 
             return result.ToString();
         }
+
+        private JObject LoadGlobalTokens(string templateDir)
+        {
+            var path = Path.Combine(templateDir, "LocalizedTokens/docs(en-us).html/tokens.json");
```

Comment:
>en-us [](start = 71, length = 5)

question: this varies with loc templates. How to support this?

Reply:
just change it to `/docs({locale}).html` when we support loc